### DISCUSSION TITLE
Not able to list some folders in root

### DISCUSSION THREAD

**Discussion by arafatx** [termux/termux-app#2739] [General]
When I go to the root directory in Termux / and then type the command ls. I got the following permission error 

`ls: cannot open directory '.' : Permission Denied`

But inside this root directory, i'm able to access other directories like `/sdcard` and listing all the files. Is this because my phone is not rooted?

I have tried this:

`chmod 755 ` . (when I'm inside the dir)

`chmod 755 /`

This will trigger the following error:

`chmod: changing permission of '.': Read-only file system`

I'm expecting to go to $ROOT and do ls, it will give me list of current folders but I got nothing but error. Accessing the sdcard that is located in root like this is working fine:

`ls $ROOT/sdcard`

**Comment by DethByte64** [comment]
Are you rooted? Are you currently root?

**Reply by arafatx** [reply]
No, phone is not rooted. I think that could be the reason. Thanks for mentioning.

**Reply by DethByte64** [reply]
It is the reason.

**Comment by agnostic-apollo** [comment]
Termux's "root" directory is `/data/data/com.termux/files`.

The android's root directory is `/` and app users can't access it fully.

https://wiki.termux.com/wiki/Differences_from_Linux
